common/Util.hpp

```diff
@@ -392,6 +393,21 @@ namespace Util
         return trimmed(std::string(s));
     }
 
+    // Trim all type of whitespace from left and right
+    inline std::string trim_whitespace(const std::string& s)
+    {
+        std::string result = s;
```

Comment:
If we will make a copy in the function anyway, why take the argument as const& ? The argument can be `std::string result` (or `s`).

Reply:
yup makes sense. My bad.